### PR TITLE
Inspector Scrolling #56

### LINKED ISSUE #56: Inspector Scrolling
The inspector needs to be able to scroll. Current content is off the screen.

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
Fixes #56 

- Add style for scroll in inspector.

### CODE REVIEW

File: src/components/vue-form-builder.vue [hunk 1]
```diff
@@ -269,7 +269,14 @@ export default {
     width: 340px;
     max-width: 340px;
     border-left: 1px solid #e9edf1;
+    overflow: hidden;
+    padding-bottom: 5%;
```
Comment (tdondich) [inline src/components/vue-form-builder.vue]: NO NO NO.  Utilize flexbox.  Do NOT use percentages like this.

File: src/components/vue-form-builder.vue [hunk 2]
```diff
@@ -269,7 +269,14 @@ export default {
...
+  }
+
+  .inspector-container > .card-body {
     overflow: auto;
+    height: 95%;
+    padding-bottom: 20px;
```
Comment (tdondich) [inline src/components/vue-form-builder.vue]: HORRIBLE.  Do not make assumptions on the overall layout dimensions.  Utilize flexbox.

File: src/components/vue-form-builder.vue (hunk 1 above)
Comment (velkymx) [reply src/components/vue-form-builder.vue]: The idea behind the bottom padding is to give a little extra spacing for the scroll.

Comment (tdondich) [reply src/components/vue-form-builder.vue]: Percentages, ideally, should be avoided as it makes unsafe assumptions regarding viewport dimensions.  You can use calc() css statements if needed to do math more accurately.  Ideally, padding should also be based on 4s.  So 4px, 8px, 16px, 32px.

**Pull request merged**